src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.test.js

```diff
@@ -270,6 +270,47 @@ describe('convertInclude', () => {
     });
   });
 
+  it('should fetch and convert an include block with a propagated renderer_env value', async () => {
+    fetch.mockResponse(() => Promise.resolve(idt1Markup));
+    const input = {
+      required: false,
+      tile: 'A quiz!',
+      href: '/indepthtoolkit/quizzes/123-456',
+      platform: 'highweb',
+      type: 'include',
+    };
+    const json = null;
+    const assetType = null;
+    const pathname = '/service/foobar?renderer_env=live';
+    const expected = {
+      type: 'include',
+      model: {
+        href: '/indepthtoolkit/quizzes/123-456',
+        required: false,
+        tile: 'A quiz!',
+        platform: 'highweb',
+        type: 'idt1',
+        html: idt1Markup,
+      },
+    };
+    expect(await convertInclude(input, json, assetType, pathname)).toEqual(
```

Comment:
Minor style suggestion, I follow [Arrange-Act-Assert](https://github.com/testdouble/contributing-tests/wiki/Arrange-Act-Assert) when I write tests, you have followed this here but I just find it nice to make the steps a little clearer so:
```javascript
    const actual = await convertInclude(input, json, assetType, pathname);

    expect(actual).toEqual(expected)
    //... rest of assertions
```

Just seperated the 'act' line with some whitespace, then proceeded with the assertions. No big deal but just think it read a little clearer to me.